### PR TITLE
* Test against Redis 3 & 4 on Travis

### PR DESCRIPTION
**Pull request by PikachuEXE**
Not sure why the test is run 3 times when running `rake` locally
But should not affect Travis build

### CODE REVIEW

**Comment by PikachuEXE** [comment]
Builds with Ruby 2.1 or below and Redis 4.x fail
Maybe they should be excluded?

**Comment by tubbo** [comment]
I think Appraisal runs the tests using the gems in `Gemfile` first, then again for each gem set in the `Appraisals` file.

**Comment by tubbo** [comment]
@PikachuEXE is there a way we can only test usage of redis 4.x against ruby 2.2 and above? they dropped support for it: https://github.com/redis/redis-rb/blob/master/CHANGELOG.md#40

File: .travis.yml [hunk 1]
```diff
@@ -6,13 +6,20 @@ script: 'bundle exec rake'
 rvm:
   - 1.9.3
   - 2.0.0
-  - 2.1
-  - 2.2
-  - 2.3.0
+  - 2.1.10
+  - 2.2.8
+  # Since the Travis Build Env does not recognize `2.3` alias yet
```
[the remark this answers is not in the document]
Comment (PikachuEXE) [reply .travis.yml]: Ya, that's an old comment which I forgot to remove
But they do use older version if an alias is used instead of exact version number

**Comment by PikachuEXE** [comment]
Excluded ruby <= 2.1 for Redis 4.x
Build is green

**Comment by tubbo** [comment]
you're a travis master @PikachuEXE, thanks for clearing up the build here and letting us test out older versions while still moving forward!

**Comment by PikachuEXE** [comment]
I learn it from other repo
Quite useful especially for gems (can test multiple versions not just on Travis but also locally)

**Pull request merged**